src/main/app/cypress/integration/komo/osaamisala.spec.js

```diff
@@ -0,0 +1,14 @@
+const autoRecord = require('cypress-autorecord');
+describe('Osaamisala KOMO', function () {
+  autoRecord();
+  it('Osaamisala KOMO renders properly', function () {
+    cy.visit('/konfo/fi/koulutus/1.2.246.562.13.00000000000000000623');
+
+    // Wait for everything to load
+    cy.findByRole('progressbar').should('not.exist');
+    cy.get('h1').contains('Hevosten kengittämisen osaamisala');
+    cy.contains('Muu Ammatillinen koulutus');
```

Comment:
Olisi hyvä pyrkiä käyttämään `testing-library?:n tarjoamia `findBy*`-rajapintoja, koska noissa Cypressin omissa rajapinnoissa on joitan omituisuuksia: https://stackoverflow.com/questions/64082074/cypress-what-is-the-difference-between-cy-contains-and-cy-findbytext